apps/channels/models.py

```diff
@@ -40,7 +40,7 @@ def _set_telegram_webhook(experiment_channel: ExperimentChannel):
     """
     Set the webhook at Telegram to allow message forwarding to this platform
     """
-    tele_bot = TeleBot(experiment_channel.extra_data.get("bot_token", ""), parse_mode=None)
+    tele_bot = TeleBot(experiment_channel.extra_data.get("bot_token", ""), threaded=False)
```

Comment:
Threaded=True will start threads that handles certain events happening inside TeleBot. We don't use that, so it's safe to not start the threads rather.